### PR TITLE
General Markdown Maintenance & Addition of Dockerfile for Easy Testing w/o Python

### PR DESCRIPTION
**Pull request by Jeymz**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [X] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [X] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [X] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [X] All your assets are stored in the **assets** folder.
- [X] All the images used are in the **PNG** format.
- [X] Any references to websites have been formatted as `[TEXT](URL)`
- [X] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue N/A.

Thank you again for your contribution :smiley:

### CODE REVIEW

**Review by szh — APPROVED** [review]
I'm super excited about this. Should make things smoother for the upcoming bug bash (#1451). Docker workflow works great.

**Comment by mackowski** [comment]
I think that changes in the CODEOWNERS file will not work. GitHub also shows the notice about that in the PR
Screenshot 2024-08-01 at 09 06 21

**Review by mackowski — CHANGES_REQUESTED** [review]
I think that changes in the CODEOWNERS file will not work. GitHub also shows the notice about that in the PR
Screenshot 2024-08-01 at 09 06 21

**Comment by jmanico** [comment]
Even though checks passed @mackowski still has concerns that seem very reasonable. What is next?

**Comment by szh** [comment]
I tried to push a commit that removes the section of the CODEOWNERS file with the error, but I'm getting "permission denied" despite the "Maintainers are allowed to edit this pull request" setting. @Jeymz can you take care of it instead?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-# commjoen have full access to Secrets_Management_CheatSheet
-* /cheatsheets_draft/Secrets_Management_CheatSheet.md @commjoen 
+- @mackowski @jmanico @szh
 
-# Kevin W. Wall (kwwall) 
-/cheatsheets/Authentication_Cheat_Sheet.md kevin.w.wall@gmail.com @mackowski @jmanico @szh
+## commjoen have full access to Secrets_Management_CheatSheet
+
+- /cheatsheets_draft/Secrets_Management_CheatSheet.md @commjoen
+
```
Comment (szh) [inline .github/CODEOWNERS:10]:
```suggestion
```

**Comment by szh** [comment]
Hmm, it seems I can do it from the web interface

**Review by szh — APPROVED** [review]

**Comment by Jeymz** [comment]
Sorry for the lack of response. I don't regularly check github since it's not my primary working repository. It looks like Shlomo was able to resolve the issues. This was not an issue with the update but was an issue with an outdated entry that was still included in the codeowners I believe. Should probably warrant a regular review of those permissions.

**Pull request merged**